pkg/api/testing/fuzzer.go

```diff
@@ -357,6 +357,11 @@ func FuzzerFor(t *testing.T, version unversioned.GroupVersion, src rand.Source)
 		},
 		func(pvc *api.PersistentVolumeClaim, c fuzz.Continue) {
 			c.FuzzNoCustom(pvc) // fuzz self without calling this function again
+			pvc.Spec.Selector = &unversioned.LabelSelector{
```

Comment:
@lavalamp there's already a TODO for it elsewhere in the file -- I would do it in a follow up if you wanted but it seemed like quite a bit of work -- this at least tests that the selelector goes through the conversion appropraitely.